code/script.js

```diff
@@ -2,220 +2,232 @@
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
+const filterButton = document.getElementById('filter')
+const winOrLose = document.getElementById('winOrLose')
+const winOrLoseText = document.getElementById('winOrLoseText')
+const playAgain = document.getElementById('playAgain')
+let winOrLoseContainer = document.getElementById('winOrLoseContainer')
+let winSound = new Audio ('./assets/win.wav');
+winSound.volume = 0.5;
+
+let loseSound = new Audio ('./assets/lose.mp3')
+loseSound.volume = 0.5;
```

Comment:
Really impressed with the sound effects and that you made them work with the conditional statement.